.rubocop.yml

```diff
@@ -13,41 +13,118 @@ Bundler/OrderedGems:
   Enabled: false
 Gemspec/OrderedDependencies:
   Enabled: false
+Layout/ClosingParenthesisIndentation:
+  Enabled: false
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
 Layout/LineLength:
+  Max: 100
+  Exclude:
+    - "spec/**/*_spec.rb"
```

Comment:
TBH I've be inclined to set the `Max` back to the `120` that is the default for rubocop. (I think the `100` value from the dry-rb rules came in when @solnic imported rules for the very first time, over 2 years ago).